build.proj

```diff
@@ -30,6 +31,61 @@
     <RemoveDir Directories="$(BaseOutputPath)" />
   </Target>
 
+  <Target Name="CopyAndDisassembleReferenceOnlyPackages"
+          AfterTargets="Build"
+          Condition="'$(OfflineBuild)' != 'true'">
+
+    <CopyReferenceOnlyPackages
+        PackageCacheDir="$(PackagesDir)"
+        DllDestinationDir="$(ReferencePackagesSourceDir)"
+        IdentifiedPackagesDir="$(ReferencePackagesToDeleteDir)"
+        DestinationDir="$(ReferencePackagesStagingDir)"
+        />
+
+    <!-- Ildasm has trouble writing to paths that are longer than 260 chars. (see https://github.com/dotnet/coreclr/issues/20397)
+         Create a temporary path to write il output which will then be copied back to the reference-packages/source dir.
+    -->
+    <PropertyGroup>
+      <TempPath>$([System.IO.Path]::GetTempPath())</TempPath>
+      <IldasmTempOutputFolderName>$([System.IO.Path]::GetRandomFileName())</IldasmTempOutputFolderName>
+      <IldasmTempOutputPath>$([System.IO.Path]::Combine($(TempPath),$(IldasmTempOutputFolderName)))/</IldasmTempOutputPath>
```

Comment:
I feel like it would be good manners to try to delete this folder when we're done, but I'm not sure... what do you think?

Reply:
Agreed. Fixed.